crates/symbolicator-service/src/services/download/s3.rs

```diff
@@ -264,6 +257,7 @@ impl S3Downloader {
     }
 }
 
+/*
```

Comment:
I believe you can leave these tests in, they should be noops in case our credentials are not set.

Reply:
I'll have to spend some time to convert the tests from rusoto to aws-sdk-rust, please stay tuned.

Reply:
The tests now compile.